src/modules/edc-demo/components/contract-agreement-transfer-dialog/model/contract-agreement-datasink-form-model.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  FormArray,
+  FormControl,
+  FormGroup,
+  ɵFormGroupValue,
+} from '@angular/forms';
+import {DataAddressType} from '../../data-address-type-select/data-address-type';
+import {HttpDatasinkAuthHeaderType} from './http-datasink-auth-header-type';
+import {HttpDatasinkHeaderFormModel} from './http-datasink-header-form-model';
+
+/* Form Model for ContractAgreementTransferDialog > DataSink */
+export interface ContractAgreementDatasinkFormModel {
```

Comment:
also this class should not exist, its contents should be placed in ContractAgreementTransferDialogFormModel